res/xml/he_il_1452_1.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard bottom_row="false">
```

Comment:
Why do you need a specific bottom row ? This adds maintenance, as the bottom row can evolve independently from the rest of the layout. For example, it changed today: 854eff2

Reply:
It was done for aesthetics reasons but those aren't relevant anymore after the recent changes so removed.